library/azure_rm_rediscache.py

```diff
@@ -570,6 +657,41 @@ def get_rediscache(self):
 
         return False
 
+    def force_reboot_rediscache(self):
+        '''
+        Force reboot specified redis cache instance in the specified subscription and resource group.
+
+        :return: True
+        '''
+        self.log("Force reboot the redis cache instance {0}".format(self.name))
+        try:
+            response = self._client.redis.force_reboot(resource_group_name=self.resource_group,
+                                                       name=self.name,
+                                                       reboot_type=self.reboot['reboot_type'],
+                                                       shard_id=self.reboot.get('shard_id'))
+        except CloudError as e:
+            self.log('Error attempting to force reboot the redis cache instance.')
```

Comment:
i wonder if we really need log and error

Reply:
why not? if user specify reboot, they should be able to know what happened if reboot failed.

Reply:
so error will do that :-)
well, not important anyway....